packages/react-component-library/src/components/DatePickerE/DatePickerE.tsx

```diff
@@ -120,6 +139,9 @@ export const DatePickerE: React.FC<DatePickerEProps> = ({
   initialMonth,
   placement = 'bottom-start',
   onBlur,
+  // Formik can pass value – drop it to stop it being forwarded to the input
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Is it worth disabling this lint rule? Or maybe there is a option to ignore `_`.  I've also use this elsewhere.

Reply:
Yep I agree on on the latter – it should support patterns, let me check if it can be set to ignore variables starting with `_`.